src/v/compat/metadata_dissemination_compat.h

```diff
@@ -100,8 +100,8 @@ struct compat_check<cluster::update_leadership_reply> {
     static void to_json(
       [[maybe_unused]] cluster::update_leadership_reply obj,
       [[maybe_unused]] json::Writer<json::StringBuffer>& wr) {}
-    static cluster::update_leadership_reply from_json([
-      [maybe_unused]] json::Value& rd) {
+    static cluster::update_leadership_reply
+    from_json([[maybe_unused]] json::Value& rd) {
```

Comment:
looks like this hunk can be squashed into a previosu commit?

Reply:
oops, I think this is from clang-format.. undone..